### PR TITLE
Add Chinese translation

### PR DESCRIPTION
**Pull request by xxnuo**
I like your style.

### CODE REVIEW

**Comment by wintermute-cell** [comment]
Thanks!

A couple points before I can merge this:
- please put both the README and the index.html file into a new `translations/` directory
- please ensure the formatting of edits and new files matches the current style. there should be no reformatting in the pull request.
- to encourage other translations in the future, I've added a `translations.html` page. please move the link to your translation to that page; similarly, there is now a new section in the README.md file for translations. in both cases, you're very welcome to add self attribution! (please see be64575c5f40feb2cb23b3b73b8f7096bef897df for more info about the new page and section)

Looking forward to merging your contribution!

**Comment by xxnuo** [comment]
🆗 Done.
btw，the icon is not displayed properly in dark mode

File: index.html [hunk 1]
```diff
@@ -18,8 +18,8 @@
 
     <!-- For the humans -->
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css">
-    <link rel="stylesheet" href="book-header.css">
-    <link rel="stylesheet" href="magick.min.css">
+    <link rel="stylesheet" href="../book-header.css">
```
Comment (wintermute-cell) [inline index.html]: I think these path changes were intended to go into `translations/zh-CN.html` ?

File: README.md [hunk 2]
```diff
@@ -5,6 +5,9 @@
 <img alt="a wizard" src="wizard.webp" width="240" height="240">
 
 ### Translations
+
+<p><a href="./translations/zh-CN.md">中文</a></p>
```
Comment (wintermute-cell) [inline README.md]: please use markdown syntax here `- [中文 (Simplified Chinese)](./translations/zh-CN.md)`

File: translations.html [hunk 3]
```diff
@@ -66,6 +66,10 @@ <h1>Translations</h1>
             by <a href="https://github.com/YOUR-USER">YOUR-USER</a>
           </li>
           -->
+          <li>
+            <a href="translations/zh-CN.html">Simplified Chinese</a>
```
Comment (wintermute-cell) [inline translations.html]: please add the native version as well: `中文 (Simplified Chinese)`

**Comment by xxnuo** [comment]
Done.
Btw the translation icon is not displayed properly in dark mode.
图片

**Review by wintermute-cell — CHANGES_REQUESTED** [review]
thanks again, just two small notes, then this is ready to merge!

File: index.html (hunk 1 above)
Comment (wintermute-cell) [reply index.html]: this is fixed already!

**Comment by wintermute-cell** [comment]
and thanks for noticing the symbol, I'll fix that!

File: index.html (hunk 1 above)
Comment (xxnuo) [reply index.html]: > I think these path changes were intended to go into `translations/zh-CN.html` ?

You can re-sync the code. I just made a mistake and now it's fixed

**Review by wintermute-cell — APPROVED** [review]

**Comment by wintermute-cell** [comment]
great, and a final thank you, merged! :)

**Pull request merged**